### ISSUE TITLE
Pod official page redirected to homepage/github source page 

### ISSUE THREAD

**Issue by dennycd** [CocoaPods/cocoapods.org#457]
Hi, one of our pod (https://cocoapods.org/pods/BoringSSL-GRPC) is always redirected to its homepage URL instead of landing on the official pod page on cocoapod. I noticed there are a number of similar reports having the same issues 

https://github.com/CocoaPods/cocoapods.org/issues/448
https://github.com/CocoaPods/cocoapods.org/issues/443

Appreciate if anyone can help look into it

**Comment by gammelby** [comment]
We have the same issue for several pods, some listed in #390 - and https://cocoapods.org/pods/NabtoEdgeClientSwift, https://cocoapods.org/pods/NabtoEdgeClientApi.

**Comment by pvenum** [comment]
We are facing the same issue as well https://cocoapods.org/pods/OTXCFramework .

**Comment by OriMedallia** [comment]
Same https://cocoapods.org/pods/MDBannerViewSDK

**Comment by narangkishore98** [comment]
Same https://cocoapods.org/pods/DotBar

**Comment by gadphly** [comment]
Any updates on this issue?  We are seeing a similar problem for our project.

**Comment by danielamitay** [comment]
This is the root cause: https://github.com/CocoaPods/cocoapods.org/blob/master/app.rb#L114-L124

Essentially, if Cocoapods is not able to fetch the Github metrics (stargazers, etc) for whatever reason, it will simply redirect to the pod's homepage.

As far as I can tell, these metrics are no longer displayed anywhere in the Cocoapods pages, so ideally this check would simply be removed, as well as the metrics fetch/anything depending on it. I could be wrong about this still being in use though.

**Comment by orta** [comment]
The metrics in this case also includes things like the rendered readme/changelog, so there wouldn't be anything to render without that info! Those are generated by https://github.com/CocoaPods/cocoapods-metadata-service - so maybe the issue lays there

**Comment by NachoSoto** [comment]
@orta looks like https://github.com/CocoaPods/cocoapods-metadata-service/pull/24 might have been the fix? 🙏🏻 
Are there any other errors that are still causing this?

**Comment by orta** [comment]
Yeah, it looks like about 20 times a day we hit this in the logs too:

```
2023-08-04T19:29:00.647873+00:00 heroku[router]: at=info method=POST path="/webhook/hooks/trunk host=cocoapods-metadata-service.herokuapp.com request_id=641f60b5-18bc-4d0a-9741-20558f4e302e fwd="18.208.148.102" dyno=web.1 connect=1ms service=2ms status=200 bytes=217 protocol=https

2023-08-04T19:29:02.132708+00:00 app[web.1]: [FunnelConnectSDK - 0.0.97] Getting info on README, CHANGELOG and community metrics.Uncaught error: error: syntax error at or near "WHERE"
```

so maybe not completely done
